package.json

```diff
@@ -101,6 +102,7 @@
     "tcp-port-used": "^1.0.1",
     "tmp": "0.0.33",
     "universal-analytics": "^0.4.16",
+    "unzipper": "^0.10.5",
```

Comment:
Can I suggest using [`archiver`](https://www.npmjs.com/package/archiver) which is already in `package.json`?

Reply:
Apparently that library only zips, does not unzip:
https://github.com/archiverjs/node-archiver/issues/97